### DISCUSSION TITLE
What's the ControlNet Unblur (XL) model?

### DISCUSSION THREAD

**Discussion by ricperry** [Acly/krita-ai-diffusion#927] [General]
I'm running a local server, and I want all the features of the standalone server. But for the life of me I can't figure out where to get the Unblur ControlNet model for SDXL.

**Comment by Acly** [comment]
Commonly known as tile models, https://huggingface.co/xinsir/controlnet-tile-sdxl-1.0 or https://civitai.com/posts/2122702

**Reply by zoomenok** [reply]
Hi, can you point what folder this safetensor should be put in?

**Reply by ricperry** [reply]
I think you can download using the manager tool. Just go into models and search xinsir or controlnet-tile and you should find it. When you download it through the manager it will place it in the expected location.
